.coveragerc

```diff
@@ -4,3 +4,9 @@ omit =
     *tests*,
     *test*,
     core/*.py
+
+[run]
```

Comment:
It seems tat the ‘run’ section has already been defined in .coveragerc. This duplicate definition is causing errors in the CI script. Let’s merge your changes in to the existing ‘run’ definition so CI will pass

Reply:
```suggestion
```